Source/WTF/wtf/CagedPtr.h

```diff
@@ -50,36 +48,31 @@ class CagedPtr {
 
     CagedPtr() : CagedPtr(nullptr) { }
     CagedPtr(std::nullptr_t)
-        : m_ptr(shouldTag ? tagArrayPtr<T>(nullptr, 0) : nullptr)
+        : m_ptr(nullptr)
     { }
 
-    CagedPtr(T* ptr, size_t size)
-        : m_ptr(shouldTag ? tagArrayPtr(ptr, size) : ptr)
+    CagedPtr(T* ptr, size_t)
+        : m_ptr(ptr)
     { }
 
-    T* get(size_t size) const
+    T* get(size_t) const
```

Comment:
Get rid of the unused `size` argument?  This also reduces code at client sites.

Reply:
Removed.